scripts/send_email.js

```diff
@@ -0,0 +1,86 @@
+// Load the AWS SDK for Node.js
+var AWS = require('aws-sdk');
+// Set the region
+var configFile = process.env.RESOURCEROOT + "/aws_ses_config.json";
+var params;
+
+AWS.config.loadFromPath(configFile);
+
+function createMessage(toEmail, subOrCur) {
+
+    // Create sendEmail params
+    if (subOrCur == "Sub") {
```

Comment:
Instead of checking the string, just make them separate functions that you export.

Make the common bottom sendEmail() code in a function that isn't exported but is called with the params for each email.

Reply:
Addressed in upcoming push